data/hai/hai reveal 4 middle.txt

```diff
@@ -1802,9 +1802,10 @@ mission "Hai Reveal [A13] A New Elder"
 				`	"What has Teeneep done, Sayari?"`
 					goto hasdone
 				`	"I take it you need another ride?"`
-			`	She puts her paw over her face and breathes a long sigh. "Yes, Captain <last>, I do need another ride." She smiles and laughs a little before looking at you again.`
+			`	She puts her paw over her face and breathes a long sigh. "Yes, Captain <last>, I do need another ride." She smiles sadly and looks away for a moment.`
 			label hasdone
-			`	She explains, "Things might be tense now, but I promise, things aren't as bad as they look. I know Teeneep, and she has the influential backing of another long-time advocate for sensible change, who was also just elected. They are remaining here to reinforce her position. Teeneep displaced that Elder who threatened her in the chambers. It's upset all the long-term power dynamics in the Council, but it's a much better Council overall now. You're to come with us too, and I'm told I'm required to travel with you, as insurance."`
+			`	Her tone is very serious as she explains, "Things might be tense now, but I promise, things aren't as bad as they look. I know Teeneep, and she has the influential backing of another long-time advocate for sensible change, who was also just elected. They are remaining here to reinforce her position. Teeneep displaced that Elder who threatened her in the chambers. It's upset all the long-term power dynamics in the Council, but it's a much better Council overall now.`
```

Comment:
Was Aimet also just elected? This part will need changing now.

Reply:
Well, we know who this 'long-time advocate' is now because he plays a big role and Sayari knows that the player knows who this is, so she shouldn't talk around Aimet like this and just say something like "Aimet is remaining here to reinforce her position".

Reply:
Ah, it's not Aimet being referred to here, but the character in question isn't identified presently.

Reply:
Hmm, perhaps what Aimet is doing now could be made clearer then. I thought that this (unnamed before) person was meant to be Aimet from the moment Aimet was given a name and a role.

Reply:
I mean, he's ancient, and very much not the sort of Hai who'd be up for flying off on a potentially military expedition to an *external* political situation. He's all about the internal mechanics.
Derpy has talked about expanding out the intro with more missions and time to build rapport, so I'd like to introduce the actual character there.
I'd also like to introduce them as the main political antagonist of the Uhai when we do the Uhai campaign tie-in, because they're a character effectively promoting a third alternative to the Elders or the Uhai